cmd/minikube/cmd/docker-env.go

```diff
@@ -427,7 +427,9 @@ docker-cli install instructions: https://minikube.sigs.k8s.io/docs/tutorials/doc
 			// TODO: refactor to work with docker, temp fix to resolve regression
```

Comment:
On line 393 there's a docker connectivity test, we should skip it if the container runtime isn't docker as it will always fail.